plugins/ltac/extratactics.mlg

```diff
@@ -482,9 +482,14 @@ TACTIC EXTEND specialize_eqs
 | [ "specialize_eqs" hyp(id) ] -> { specialize_eqs id }
 END
 
+{
+let warn_destauto = CWarnings.create ~name:"destauto-deprecated" ~category:Deprecation.Version.v8_20
+    (fun () -> strbrk "The \"destauto\" tactic is deprecated.")
+}
+
 TACTIC EXTEND destauto
```

Comment:
There is a DEPRECATED syntax precisely for this situation, see e.g. the recent deprecation of gintuition to get an example.

Reply:
Got it.